frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -75,9 +77,22 @@ export default class Dashboard extends Component {
     isSharing: false,
   };
 
+  constructor(props) {
+    super(props);
+    this.parametersWidgetRef = React.createRef();
+    this.parametersAndCardsContainerRef = React.createRef();
+  }
+
   // NOTE: all of these lifecycle methods should be replaced with DashboardData HoC in container
   componentDidMount() {
     this.loadDashboard(this.props.dashboardId);
+
+    window.addEventListener("scroll", this.updateParametersWidgetStickiness, {
```

Comment:
Is there any value of using `_.throttle` here?
I seem to recall that this was necessary for some pop-up resizing. /cc @daltojohnso

Reply:
@ariya @daltojohnso 

I will experiment with throttling, thanks for the suggestion.

Reply:
Done. 

Feels like a trade off:

for all browsers but FF it makes things very slightly worse from the best baseline.

for FF it makes things a heck of a lot better from a very low baseline.

Keeping the throttle at 24fps for now.

Reply:
Great job at identifying the potential issue with Firefox scrolling!